src/components/List/ListItem/ListItem.tsx

```diff
@@ -19,6 +19,8 @@ interface Props extends Omit<React.HTMLAttributes<HTMLLIElement>, 'onClick'> {
     highlightContext?: Context | 'default';
     /** Ref to access the li element */
     ref?: React.RefObject<HTMLLIElement>;
+    /** Item identifier */
+    value?: string | number | boolean | object;
```

Comment:
This should also be destructured from props, so that it is not assigned to the `li` element. Otherwise we might make incorrect HTML, since `value` on `li` should be an integer (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li)

Reply:
It should be integer only for ordered lists `<ol>` which we don't use, but yes we don't know for sure how this component can be used =)
If one day we need to read it from element we can use data attribute.

---

> I think unknown type is better fits for the value that can have different types

agree, just `unknown` looks weird for me